python2/test_typing.py

```diff
@@ -1110,6 +1111,36 @@ def __len__(self):
         self.assertIsSubclass(MMC, typing.Mapping)
 
 
+class TypeTests(BaseTestCase):
+
+    def test_type_basic(self):
+
+        class User(object): pass
+        class BasicUser(User): pass
+        class ProUser(User): pass
+
+        def new_user(user_class):
+            # type: (Type[User]) -> None
```

Comment:
@gvanrossum This may be a stupid question but shouldn't this (and https://github.com/python/typing/pull/224/files#diff-f9eeb4530dadc70935c11fff001ec7acR1385) be marked as returning ``User``?